src/config/app.config.test.ts

```diff
@@ -129,6 +129,56 @@ describe('appConfig', () => {
     });
   });
 
+  describe('parsing SINGLE_LINE_LOG_FORMAT', () => {
```

Comment:
could we use `withEnvironmentVariableParsingUnitTests` here at all?

Reply:
@AlexBramhill so the notifications go to the right person

Happy for this to be a separate PR if you want

Reply:
Ill add it in this one -- two minutes

Reply:
I'll add this in a new PR -- as it'll be for all app.config things :)